src/components/commons/Header.tsx

```diff
@@ -104,7 +105,13 @@ export default function Header() {
                   onClick={() => toggleDrawer(true)}
                 >
                   {moneyFormat(amountToShow())}
-                  <InfoOutlinedIcon color="primary" sx={{ ml: 1 }} />
+                  <img
+                    src={cartIcon}
+                    alt="Icona carrello"
```

Comment:
is the alt text right in Italian language ?

Reply:
I've assumed that it's ok, you can find other imgs with the same approach. Like [here](https://github.com/pagopa/pagopa-checkout-fe/blob/aa69616a71fbb225fbfdf3e16f684033d557996f/src/components/TextFormField/PspFieldContainer.tsx#L56)